src/name/admitriev/jhelper/configuration/TaskConfiguration.java

```diff
@@ -166,6 +167,7 @@ private static Test readTest(Element element) {
 
 	@Override
 	public void writeExternal(Element element) {
+		element.setAttribute("name", getName());
```

Comment:
My understanding is that name is already saved automatically. Can you please explain a bit more how it's used?

```xml
<component name="ProjectRunConfigurationManager">
  <configuration default="false" name="A1: Consistency - Chapter 1" type="name.admitriev.jhelper.configuration.TaskConfigurationType" className="A1ConsistencyChapter1" cppPath="tasks/A1ConsistencyChapter1.cpp" inputType="LOCAL_REGEXP" inputFile="consistency_chapter_.*input[.]txt" outputType="CUSTOM" outputFile="consistency_chapter__output.txt" testType="MULTI_NUMBER">
    <tests>
      <test input="6&#10;ABC&#10;F&#10;BANANA&#10;FBHC&#10;FOXEN&#10;CONSISTENCY&#10;" output="Case #1: 2&#10;Case #2: 0&#10;Case #3: 3&#10;Case #4: 4&#10;Case #5: 5&#10;Case #6: 12&#10;" active="true" />
    </tests>
    <method v="2" />
  </configuration>
</component>```